tests/python/pants_test/engine/test_scheduler.py

```diff
@@ -214,9 +214,27 @@ def test_get_type_match_failure(self):
       self.scheduler.product_request(A, [Params(TypeCheckFailWrapper(A()))])
 
   def test_trace_includes_rule_exception_traceback(self):
+<<<<<<< Updated upstream
```

Comment:
You'll probably want to remove these conflict markers! Does there need to be any change here?

Reply:
Danny, thank you! This is not my change.